src/Neo.VM/Types/ByteString.cs

```diff
@@ -85,7 +85,7 @@ internal bool Equals(StackItem? other, ref uint limits)
         public override bool GetBoolean()
         {
             if (Size > Integer.MaxSize) throw new InvalidCastException();
-            return Unsafe.NotZero(GetSpan());
+            return GetSpan().NotZero();
```

Comment:
The implementation is quite straightforward, but `GetBoolean` is frequently-used in VM, thus we have to check mainnet/testnet networks for compatibility. There should be no issues, but let's do it just in case.

Reply:
@superboyiii could you check it?